mobidam-sst-management-frontend/frontend/src/views/MainView.vue

```diff
@@ -61,7 +61,8 @@
                     <v-tooltip location="top">
                         <template #activator="{ props }">
                             <v-col
-                                cols="2"
+                                cols="3"
+                                md="3"
                                 v-bind="props"
                             >
                                 {{ schnittstelle.name }}
```

Comment:
![grafik](https://github.com/user-attachments/assets/dd41660c-97e5-4a63-90a4-0e37c970b82e)
Theres plenty of room at the end of the table which could be better used for the sst-name

Reply:
Maybe i misunderstand this, but there is another colum.
![image](https://github.com/user-attachments/assets/ddd78ffc-aa7b-41b2-b476-3913af50e948)